spec/support/coverage_helper.rb

```diff
@@ -0,0 +1,43 @@
+module Spec
+  module CoverageHelper
+    GEM_ROOT = Pathname.new(__dir__).join("../..").expand_path
+
+    class << self
+      attr_accessor :coverage_counter
+    end
+    self.coverage_counter = 0
+
+    def self.fix_coverage_resultset_paths
+      file = GEM_ROOT.join("coverage/.resultset.json")
+      contents = File.read(file)
+      contents.gsub!(%r{(?<=").+/\.bundle/plugin/bundler/gems/bundler-inject-\h+}, GEM_ROOT.to_s)
+      File.write(file, contents)
+    end
+
+    def self.dependency_paths(gem_name)
```

Comment:
Does an arg even make sense here?  Won't it always be `"simplecov"`?

Reply:
Oh, nevermind, I see this is being used recursively.  Maybe add a note about that? (up to you)

Reply:
Though maybe `def self.dependency_paths(gem_name = "simplecov")` since this is meant for `simplecov` specifically, then the caller down in `.simplecov_dependency_paths` can just be dependency paths.

_(@NickLaMuro is probably making a bigger deal out of this than he needs to...)_

Reply:
yeah I like the default parameter

Reply:
I'm gonna leave it as is...method-name wise, I feel like the current way is cleaner.